renaming_aps/README.md

```diff
@@ -0,0 +1,78 @@
+# Renaming APs
+This is a Python script that uses the [Pycentral](https://pypi.org/project/pycentral/) library to achieve the following steps on an Aruba Central account- 
+1. Copy device configuration
+2. Change device hostname
+3. Replace old device configuration with newly named configuration
+
+## Prerequisite
+1. All devices have valid Central licenses before the workflow is run
+2. Devices are all online and configurable
+3. Devices aren't offline or not able to be configured
```

Comment:
You can remove the 3 point. I think the second point about devices being online covers that.